sstables/mc/writer.cc

```diff
@@ -933,6 +936,7 @@ void writer::drain_tombstones(std::optional<position_in_partition_view> pos) {
 }
 
 void writer::consume_new_partition(const dht::decorated_key& dk) {
+    _parquet_writer->consume_new_partition(dk);
```

Comment:
There's probably a better place for intercepting this call and delegating to parquet_writer.

Reply:
Fine for now.